x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -187,7 +182,16 @@ export const ExecuteActionRequestSchema = {
           }
         },
       }),
-      timeout: schema.maybe(schema.number({ min: 1 })),
+      timeout: schema.maybe(schema.number({ min: 0 })),
     }),
   }),
 };
+
+export type ExecuteActionRequestBody = TypeOf<typeof ExecuteActionRequestSchema.body>;
+
+export const ResponseActionBodySchema = schema.oneOf([
```

Comment:
is this correct? to include the other GetFile and Execution here since they are behind feature flags?  The only reason I'm questioning it is because `get-file` was not already int his `onfOf()` array.

Reply:
Yeah, this is correct. It seems that we didn't update this schema earlier. I had to add `ExecuteActionRequestSchema.body` to this else TS was not picking up all the different parameters [defined here](https://github.com/elastic/kibana/blob/59468bc435c28d1b3fe95b14b708b3f17c4cbac0/x-pack/plugins/security_solution/common/endpoint/types/actions.ts#L157) at [this place](https://github.com/elastic/kibana/pull/150202/files#diff-85628b4067d4181d7e1634fd7bb273a1c6a6eba8afb0e230677b54e3dfed5c8aR261) where I was trying to get the timeout.

Basically, it was only resolving to `undefined` | `ResponseActionParametersWithPidOrEntityId` earlier to this change.